using_images/s2i_images/php.adoc

```diff
@@ -120,8 +120,13 @@ The following environment variable sets its equivalent property value in the opc
 |Variable name |Description |Default
 
 |`*OPCACHE_MEMORY_CONSUMPTION*`
-|The OPcache shared memory storage size.
+|The link:http://php.net/manual/en/book.opcache.php[OPcache] shared memory storage size.
```

Comment:
I could understand(a bit) why to link the official docs fo OPcache ... but what you suggest is to basically have links all around our docs, to explain all the language specific things. 

Why don't we have any links for docs in `DISABLE_COLLECTSTATIC` or `PERL_APACHE2_RELOAD` ?